src/fstests/runner.py

```diff
@@ -41,6 +50,13 @@ def run(self):
             self._db.unsubscribe(sub_id)
             return True
 
+class cmd_check(Command):
+    help = 'Check KVM-xfstests environment'
+    args = [Args.db_config]
+    opt_args = [Args.verbose]
+
+    def __call__(self, configs, args):
+        return FstestsRunner(configs, args).check_kvm_xfstest_env()
```

Comment:
The `check_kvm_xfstests_env()` method is not returning anything, so please make it return `True` or `False` when there was an error.

Reply:
good point, I forgot to include this during the rebase.